packages/@react-spectrum/list/src/styles.css

```diff
@@ -403,8 +401,8 @@
 
     .react-spectrum-ListViewItem-grid {
       grid-template-areas:
-        "icon image content     . badge"
-        "icon image description . badge";
+        "thumbnail content     . badge"
+        "thumbnail description . badge";
```

Comment:
note for anyone coming through, the `grid-template-columns` was already correct, looks like we forgot to remove a grid area though, same for the change above